### DISCUSSION TITLE
How to use replacement_sdts with PIT macro?

### DISCUSSION THREAD

**Discussion by sti0** [ScalefreeCOM/datavault4dbt#261] [Q&A]
How to use `replacement_sdts` with PIT macro?

Currently we can specify `replacement_sdts` as a column in the pit macro (parameter `sdts`). But if we do so, the column in the pit table is also named `replacement_sdts` instead of `sdts`.

In my opinion there is a missing parameter `sdts_alias` to use the `replacement_sdts` with pit tables.

**Comment by tkiehn** [comment]
Hi @sti0 and thanks for reaching out!

You are right that this can be confusing.
I will bring up internally how to best deal with this, but I think we will implement a feature like you suggested.

E.g. a new optional parameter `replacement_sdts` which will be used in the JOIN if present, else the `sdts` is used.

Also you are welcome to contribute so feel free to open a pull request with your proposed changes.

Best Regards
Theo
